CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - ROX-9435: Updated dryrun API to generate preview violations for disabled policies
 - Support for security policies that do not have a policyVersion or have versions prior to 1.1 will be removed. If you have externally stored older policies, they cannot be imported.
 - ROX-10021: RHCOS node support is dropped until major improvements are made in ROX-8944.
+- ROX-8533: Show Cluster Health Info if Scanner is running on a secured cluster
```

Comment:
I don't think this entry is needed since it's completely captured by a ticket. I tend to only add an entry if there is either no ticket and this adds something worth documenting (for the docs team) or there is a ticket and there is some nuance to the implementation that may not be clear in the ticket. I am guilty of the entry above this `- ROX-10021: RHCOS node support is dropped until major improvements are made in ROX-8944.`, but the purpose of this entry it to let docs people know there are followup plans for this. Also, there is a followup entry which appends information about the UI (which is not really mentioned in the ticket) to the entry.

Reply:
Thanks for the explanation. Entry removed.